github_actions/lib/dependabot/github_actions/file_fetcher.rb

```diff
@@ -5,23 +5,23 @@
 
 require "dependabot/file_fetchers"
 require "dependabot/file_fetchers/base"
+require "dependabot/github_actions/package_manager"
 
 module Dependabot
   module GithubActions
     class FileFetcher < Dependabot::FileFetchers::Base
       extend T::Sig
       extend T::Helpers
 
-      FILENAME_PATTERN = /\.ya?ml$/
-
       sig { override.params(filenames: T::Array[String]).returns(T::Boolean) }
       def self.required_files_in?(filenames)
-        filenames.any? { |f| f.match?(FILENAME_PATTERN) }
+        filenames.any? { |f| f.match?(MANIFEST_FILE_PATTERN) }
       end
 
       sig { override.returns(String) }
       def self.required_files_message
-        "Repo must contain a .github/workflows directory with YAML files or an action.yml file"
+        "Repo must contain a #{WORKFLOW_DIRECTORY} directory with YAML files or " \
+          "an #{MANIFEST_FILE_YML} file"
```

Comment:
🤔 So on the one hand abstractions are nice and forward thinking, but on the other hand I an not really sure that workflows will _ever_ be allowed to run from a location beyond `.github/workflows`... If that's true, then _not_ having the abstraction is actually more clear when reading the code. Preparing for a possible future where they reside in another folder seems like premature optimization unless there's some other reason to have a constant here...